### PR TITLE
Add Rumble Icon

### LINKED ISSUE #8495: Request icon: Rumble
### Brand Name

Rumble

### Website

https://rumble.com/

### Popularity Metric

The similarweb rank is 439 (global) and 168 (country - United States).

### Official Resources for Icon and Color

SVG from website:

- https://rumble.com/img/rumble-full-logo-v4.svg
- https://rumble.com/img/rumble-full-logo-v4-dark.svg

The logo mark is green: #74CC1D

The text is dark blue, if not using dark mode: #061726

The text is white, if using dark mode: #fff

### Additional Comments

_No response_

### Contributing

Yes

### LINKED ISSUE #9214: Rumble
### Brand Name

Rumble

### Website

https://rumble.com/

### Popularity Metric

The Similarweb site for Rumble is https://www.similarweb.com/website/rumble.com/

### Official Resources for Icon and Color

https://commons.wikimedia.org/wiki/File:Rumble_logo_2022.svg

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by RiteshK-611**
**Issue:** closes #8495

**Similarweb rank:**
The Similarweb site for Rumble is https://www.similarweb.com/website/rumble.com/

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Preview
rumble

### CODE REVIEW

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Review by PratyushJayachandran — APPROVED** [review]
Seems good 😊

**Comment by RiteshK-611** [comment]
> Seems good 😊

Then who is gonna Merge it?

**Comment by PratyushJayachandran** [comment]
> Then who is gonna Merge it?

I cannot. I don't have permissions😅

**Comment by RiteshK-611** [comment]
> > Then who is gonna Merge it?
> 
> I cannot. I don't have permissions😅

No problem 🙂
Can you add reviewer who can merge it?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10850,6 +10850,11 @@
             "hex": "FCC21B",
             "source": "https://github.com/charliermarsh/ruff/blob/e04ef423344837c916e75a7b09ea674711a104e0/assets/badge/v1.json"
         },
+        {
+            "title": "Rumble",
+            "hex": "74CC1D",
```
Comment (adamrusted) [inline _data/simple-icons.json:10855]:
```suggestion
            "hex": "85C742",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon itself is spot on, just the one change needed to the hex code to match the one found in the SVG.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @RiteshK-611 🎉

**Pull request merged**